terra-app-chart/templates/deployment.yaml

```diff
@@ -24,7 +24,7 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      serviceAccountName: {{ include "app.serviceAccountName" . }}
```

Comment:
This function was deleted in https://github.com/DataBiosphere/terra-app/pull/4.. I think we can just use the value directly -- assume Leo will always pass in the service account name.